### ISSUE TITLE
New approach to video animation

### ISSUE THREAD

**Issue by NikitaBogoslovskiy** [NikitaBogoslovskiy/AvatarAnimation#12]
Now we have the problem of facial jitter because current neural network works with input landmarks independently, there is no connection between neighbouring frames. The idea is to implement neural network which will take into account previous frames and corresponding landmarks. It will require usage of RNN or LSTM layers, new loss function and new method for dataset generation that will be creating head model samples sequentially (we used to sample random parameters, now we can sample random deviations from previous parameters).
